src/main/java/net/minecraftforge/common/config/Configuration.java

```diff
@@ -80,9 +89,16 @@
 
     public Configuration(){}
 
+    @Deprecated
+    public Configuration(CharSource staticSource, CharSink staticSink){
```

Comment:
You can implement CharSource however you like. CharSource is simply something that produces a Reader. So if you want to make it read from something that the server sent you, just make it return a StringReader, a Reader that reads from a String.